seqr/management/tests/reload_clinvar_all_variants_tests.py

```diff
@@ -0,0 +1,310 @@
+        KeyLookupSnvIndel.objects.using('clickhouse_write').create(
+            variant_id='22-28695219-G-A',
+            key_id=12,
+        )
+        KeyLookupMito.objects.using('clickhouse_write').create(
+            variant_id='M-123-GA',
+            key_id=8,
+        )
+        ClinvarAllVariantsSnvIndel.objects.using('clickhouse_write').create(
+            version='2025-06-23',
+            variant_id='22-28695219-G-A',
+            allele_id=20642,
+            pathogenicity='Likely_pathogenic',
+            assertions=[],
+            conflicting_pathogenicities=[],
+            gold_stars=2,
+            submitters=[],
+            conditions=[]
+        )
+        DataVersions('Clinvar', '2025-06-23').save()
+        responses.add(responses.GET, WEEKLY_XML_RELEASE, status=200, body=gzip.compress(WEEKLY_XML_RELEASE_DATA.encode()), stream=True)
+        call_command('reload_clinvar_all_variants')
+        mock_logger.assert_called_with('Updating Clinvar ClickHouse tables to 2025-06-30 from 2025-06-23.')
+        seqr_clinvar_snv_indel_models = ClinvarSnvIndel.objects.all()
+        self.assertEqual(seqr_clinvar_snv_indel_models.count(), 1)
+        self.assertDictEqual(
+            model_to_dict(seqr_clinvar_snv_indel_models.first()),
+            {
+                'allele_id': 20642,
+                'assertions': ['association', 'risk_factor'],
+                'conditions': [
+                    'Breast carcinoma',
+                    'CHEK2-related cancer predisposition',
+                    'CHEK2-related disorder',
+                    'Hereditary cancer-predisposing syndrome',
+                ],
+                'conflicting_pathogenicities': [
+                    {'pathogenicity': 'Pathogenic', 'count': 19},
+                    {'pathogenicity': 'Likely_pathogenic', 'count': 9},
+                    {'pathogenicity': 'Established_risk_allele', 'count': 1},
+                    {'pathogenicity': 'Likely_risk_allele', 'count': 1},
+                    {'pathogenicity': 'Uncertain_significance', 'count': 1},
+                ],
+                'gold_stars': 1,
+                'key': 12,
+                'pathogenicity': 'Conflicting_classifications_of_pathogenicity',
+                'submitters': [
+                    'Fulgent Genetics, Fulgent Genetics',
+                    'Revvity Omics, Revvity',
+                    'University of Washington Department of Laboratory Medicine, University of Washington'
+                ]
+            },
+        )
+
+        seqr_clinvar_grch37_snv_indel_models = ClinvarGRCh37SnvIndel.objects.all()
+        self.assertEqual(seqr_clinvar_grch37_snv_indel_models.count(), 0)
+        clinvar_all_variants_grch37_snv_indel_models = ClinvarAllVariantsGRCh37SnvIndel.objects.all()
+        self.assertEqual(clinvar_all_variants_grch37_snv_indel_models.count(), 1)
+        clinvar_all_variants_mito_models = ClinvarAllVariantsMito.objects.all()
+        self.assertEqual(
```

Comment:
`assertListEqual` would be better here, if this ever breaks the diff/error message is better the more specific of an equality function you use